.babelrc

```diff
@@ -4,14 +4,17 @@
       "pragma": "createElement"
     }],
     ["@babel/preset-env", {
+       "bugfixes": true,
        "targets": {
-         "browsers": ["> 0.3%"]
+         "chrome": "55",
```

Comment:
We have this notion of supporting a rolling set of browsers based on release cycles. It's expressed in various ways in various places in our apps/environments. I'm starting to wonder if we need some sort of lightweight management/checklist of places to update semi-regularly over time...any thoughts?

Reply:
I think that's a good idea. Any preferences on where to maintain it?

Reply:
Isn't that the $64 question lately...I'll think about it.